### DISCUSSION TITLE
What is this app

### DISCUSSION THREAD

**Discussion by Kevsosmooth** [pear-devs/pear-desktop#2473] [General]
Somehow I downloaded this app trying to learn about RiMusicApp and got here somehow. But is this any different from that. Am I getting a better uncompressed quality that youtube normally would give? Does this page have a reddit page a discord anything with a community?

**Comment by Lockszmith-GH** [comment]
Not sure whether you're a troll or not, I'll assume you're not, and just lacking tact.

It's a Windows/Linux/MacOS desktop app for YouTube Music, with built-in extensions you can turn on/off.

The GH discussion is where I've seen some conversations about the app happening, does it need something else?
RiMusicApp is an android app, this is a Windows app.

It can't provide anything better than YouTube, as it is just a front end for YouTube.

Hope this answers your curiosity.

**Reply by Kevsosmooth** [reply]
I was not trolling to be honest I do not see much information on the github page so I can only assume what I see. Now the reason why I mentioned RiMusic because I was told you get a less compressed version in other words a better quality. But I am using this and have noticed that it works pretty much just wanted some information on it that I previously did not have. I googled rimusic and landed here so what I am saying it was on accident lol.

**Reply by Lockszmith-GH** [reply]
All fine. Keep in mind that any project providing alternative experiences to web based services like YouTube Music are in a gray area.
Officially they shouldn't exist, and when they get too popular, they usually die of over-exposure.

Finding these are word-of-mouth or plain accidental stumbling, like you just did. Like I did a few months ago.

What I really like about this project, it relies on the existing web interface, making its modifications only that - modifications. It doesn't change the way the service is reached on the backend substantially.

**Comment by Kevsosmooth** [comment]
Yes true to be honest I literally just let the playlist run all day long lol. By any chance do you know where I would be able to find a high quality output as far as much goes? I just got new surround sounds and they were not that cheap but really I want to see if there is any difference in higher quality music vs I guess youtube quality which I hear is compressed to a certain extent.

**Reply by Lockszmith-GH** [reply]
If you're looking for lossless options, then YouTube isn't the service, neither is Spotify.
Deezer comes to mind.
